pom.xml

```diff
@@ -1,9 +1,10 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.36</version><!-- Built against the Jenkins version specified in properties as 'jenkins.version' -->
+        <version>2.11</version><!-- Built against the Jenkins version specified in properties as 'jenkins.version' -->
```

Comment:
Even more why? 👎

Reply:
Thanks for asking. As I've said above I am not Java pro so here is what happened: I tried to build it with Netbeans and with 2.36 and got this:
```
--- maven-enforcer-plugin:3.0.0-M1:enforce (display-info) @ Office-365-Connector ---
Ignoring requireUpperBoundDeps in com.google.guava:guava
Ignoring requireUpperBoundDeps in com.google.code.findbugs:jsr305
Rule 0: org.apache.maven.plugins.enforcer.RequireMavenVersion failed with message:
3.1.0 required by frontend-maven-plugin at least.
------------------------------------------------------------------------
BUILD FAILURE
------------------------------------------------------------------------
Total time: 8.958s
Finished at: Fri Dec 01 09:42:21 EET 2017
Final Memory: 53M/494M
------------------------------------------------------------------------
Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M1:enforce (display-info) on project Office-365-Connector: Some Enforcer rules have failed. Look above for specific messages explaining why the rule failed. -> [Help 1]

To see the full stack trace of the errors, re-run Maven with the -e switch.
Re-run Maven using the -X switch to enable full debug logging.

For more information about the errors and possible solutions, please read the following articles:
[Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```
Then I googled for some solutions and one of suggestions somewhere was to use lower version here. If you can help me to understand how I can make it build with 2.36 - I will be more than happy

Reply:
Figured out how to update maven. Corrected this back to 2.36